adrs/2023-02-02-automate-runner-updates.md

```diff
@@ -0,0 +1,26 @@
+# Automate updating runner version
+
+**Status**: Proposed
+
+## Context
+
+When a new [runner](https://github.com/actions/runner) version is released, new
+images need to be built in
+[actions-runner-controller/releases](https://github.com/actions-runner-controller/releases).
+This is currently started by the
+[release-runners](https://github.com/actions/actions-runner-controller/blob/master/.github/workflows/release-runners.yaml)
+workflow, although this only starts when the set of file containing the runner
+version is updated (and this is currently done manually).
```

Comment:
One thing I don't like at all about our current runner release process is the need to update all these files: https://github.com/actions/actions-runner-controller/pull/2182 

I'm not sure why all these values are needed. I'm adding this here for full awareness.

Reply:
Could we also add the alternatives and why we dismissed them?

Reply:
These changes are needed because what actions-runner-controller/releases does is pull this ARC repo and [use these Dockerfiles to build images](https://github.com/actions-runner-controller/releases/blob/main/.github/workflows/release-runners.yaml#L100-L122).

Reply:
Oh sorry I wasn't clear, what I meant to say is: I'm not confident that this should be the way to go moving forward. We should be able to pass all these arguments at runtime! I know how they're used because I migrated the pipelines :)